apps/meteor/server/lib/logger/startup.ts

```diff
@@ -1,7 +1,6 @@
+import { setQueueLimit, logLevel, type LogLevelSetting } from '@rocket.chat/logger';
+
 import { settings } from '../../../app/settings/server';
-import type { LogLevelSetting } from './logLevel';
-import { logLevel } from './logLevel';
-import { setQueueLimit } from './logQueue';
```

Comment:
I wonder if the "log queue" feature should be kept on meteor app.. it is probably not relevant to other packages/apps using the logger

Reply:
I moved it there to ensure it runs before the main pino instance is instantiated - so Pino can detect that `process.stdout.write` has been modified.